server/src/internalClusterTest/java/org/elasticsearch/cluster/coordination/RareClusterStateIT.java

```diff
@@ -201,7 +201,7 @@ public void testDeleteCreateInOneBulk() throws Exception {
                     .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
                     .put(IndexMetadata.SETTING_WAIT_FOR_ACTIVE_SHARDS.getKey(), "0")
             ).setTimeout("0s")
-        ).get(10, TimeUnit.SECONDS);
+        ).get(30, TimeUnit.SECONDS);
```

Comment:
I also see several other cases of `executeAndCancelCommittedPublication()` in this test that ultimately use `.get()` with 10 seconds. Would it be reasonable to change them as well?

Reply:
Also, any hint as to why it takes so long >10 seconds sometimes?

Reply:
To clarify, I meant if you could search for the usages of `executeAndCancelCommittedPublication` and go through the subsequent .get()s to make them use 30 seconds as well. I saw like around 5 instances.